extensions/amp-viewer-integration/0.1/test/test-highlight-handler.js

```diff
@@ -409,57 +412,79 @@ describes.realWin.skip(
       expect(setScrollTopStub).to.be.calledOnce;
       expect(setScrollTopStub.firstCall.args[0]).to.equal(350);
     });
+  }
+);
 
+describes.realWin(
+  'HighlightHandler',
+  {
+    // We can not overwrite win.location with realWin.
+    amp: {
+      ampdoc: 'single',
+    },
+  },
+  (env) => {
     // TODO(dmanek): remove `ifChrome` once we remove Chrome version detection
```

Comment:
What is this TODO about?: Is it time?

Reply:
Not yet. See: https://github.com/ampproject/amphtml/blob/main/extensions/amp-viewer-integration/0.1/highlight-handler.js#L151-L157

Reply:
I don't understand when we could ever remove the version detection

Reply:
Yes. I've been in touch with the Chrome team who are working on making the highlight "sticky" (persist through click or scroll). Once that has landed in Chrome, we can wait a few versions and remove this check.